src/renderer/views/Channel/Channel.js

```diff
@@ -352,6 +360,31 @@ export default Vue.extend({
         }
       })
     },
+    getChannelShortsLocal: function () {
+      this.isElementListLoading = true
+      const expectedId = this.originalId
+      ytch.getChannelShorts({ channelId: this.id, channelIdType: this.idType, sortBy: this.videoSortBy }).then((response) => {
+        if (expectedId !== this.originalId) {
+          return
+        }
+
+        this.latestShorts = response.items
+        this.videoContinuationString = response.continuation
```

Comment:
Should probably add a separate variable for storing the shorts continuation

Reply:
✅